src/api.js

```diff
@@ -1,6 +1,17 @@
 module.exports = class DanceAPI {
   constructor(nativeAPI) {
     const sprites = [];
+    
+    const lookupSprite = spriteName =>
+      sprites.find(sprite => sprite.name === spriteName);
+
+    const enforceUniqueName = name => {
+      sprites.forEach(sprite => {
+        if (sprite.name === name) {
```

Comment:
Can you confirm my understanding - if I create a sprite 'dog1' and create a second sprite 'dog1', this just 'removes' a way to reference the first sprite (so it gets basically overridden)?

Reply:
Yep, that's right. This is primarily to match the current behavior, though I could see at some point wanting to allow multiple sprites having the same name.